doc/schemas/invoicerequest.request.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "amount",
+    "description"
+  ],
+  "added": "v22.11",
+  "properties": {
+    "amount": {
+      "type": "msat",
+      "description": ""
+    },
+    "description": {
+      "type": "string",
+      "description": ""
+    },
+    "issuer": {
+      "type": "string",
+      "description": ""
+    },
```

Comment:
missing the `label` field here? `json_invoicerequest` looks for it, but missing from the schema.

Reply:
Yep I can confirm that I receive an error when trying to pass in a label to the `invoicerequest` method.